mediapipe/calculators/geti/scripts/setup_opencv.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+export OPENCV_VERSION=4.3.0
+
```

Comment:
This script seems to be duplicated

Reply:
It is. I copied it over from our repo and apparently didn't think too careful about this.

Reply:
Removing scripts.